platformics/api/core/helpers.py

```diff
@@ -164,6 +164,14 @@ def convert_where_clauses_to_sql(
                     query = query.filter(getattr(sa_model, col).is_(None))
                 else:
                     query = query.filter(getattr(sa_model, col).isnot(None))
+            elif sa_comparator == "not_regexp_match":
```

Comment:
`not_regexp_match` isn't a possible value anymore!

Reply:
🤦 Good catch, cleaned it up!